doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -4,21 +4,20 @@
 
 .. index:: mclock; configuration
 
-Mclock profiles mask the low level details from users, making it
-easier for them to configure mclock.  
+Dmclock-related configuration parameters can be applied by using mclock
```

Comment:
Didn't we finalize this section in your earlier PR https://github.com/ceph/ceph/pull/40570/commits/e262e85606c9e4090454e902b5102954b8020313#diff-ec476ef02ccd9d5bdec6c63b3b4fb2eb6f001090a9fd2f6a59ad9d40403cb548R5-R21?

Reply:
We did.

I've attempted to solve this problem by running these commands:

git checkout wip-doc-mclock-rewrite-second-half-2021-apr-2
git merge --patch master doc/rados/configuration/mclock-config-ref.rst